pkg/apis/build/v1alpha1/buildrun_types.go

```diff
@@ -166,7 +171,7 @@ type BuildRef struct {
 	Name string `json:"name"`
 	// API version of the referent
 	// +optional
-	APIVersion string `json:"apiVersion,omitempty"`
+	APIVersion *string `json:"apiVersion,omitempty"`
```

Comment:
Optional string where we are not required to distinguish `nil` and `""`. See discussion above.